#! /bin/sh -e

if [ $# -eq 3 -a "$2" = '-d' ]; then
    pdir="-d $3"
elif [ $# -ne 1 ]; then
    echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
    exit 1
fi
case "$1" in
    -patch) patch $pdir -f --no-backup-if-mismatch -p1 < $0;;
    -unpatch) patch $pdir -f --no-backup-if-mismatch -R -p1 < $0;;
    *)
	echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
	exit 1
esac
exit 0

# DP: use EDITOR environment variable or else the editor command.

--- ./support/bashbug.sh~	Tue Mar 17 20:21:00 1998
+++ ./support/bashbug.sh	Sun Nov 14 13:49:26 1999
@@ -49,7 +49,7 @@
 
 BUGADDR="${1-$BUGBASH}"
 
-: ${EDITOR=emacs}
+: ${EDITOR=editor}
 
 : ${USER=${LOGNAME-`whoami`}}
 
